tools/component-builder/css/lib/varUtils.js

```diff
@@ -38,6 +38,23 @@ function getVarsFromCSS(css) {
   return variableList;
 }
 
+function getVarsDefinedInCSS(css) {
```

Comment:
This gets the list of all variables defined within the component's CSS; i.e. the `--spectrum-actionbutton-height` which is based on `--spectrum-actionbutton-m-height` for the medium size, etc. This makes it so we don't throw when the variable has been defined inside of the component, instead we warn.